README.markdown

```diff
@@ -5268,7 +5268,7 @@ Note that while this method accepts all [HTTP status constants](#http-status-con
 
 Also note that this method call terminates the processing of the current request and that it is recommended that a coding style that combines this method call with the `return` statement, i.e., `return ngx.exit(...)` be used to reinforce the fact that the request processing is being terminated.
 
-When being used in the contexts of [header_filter_by_lua](#header_filter_by_lua) and
+When being used in the contexts of [header_filter_by_lua](#header_filter_by_lua), [balancer_by_lua](#balancer_by_lua) and
```

Comment:
There is no `balancer_by_lua` directive so this link is broken. It should point to the `balancer_by_lua_block` section instead.

Also, this `README.markdown` was automatically generated; do not edit it manually. Edit the `doc/HttpLuaModule.wiki` file instead.

I'll fix these on my side for you.